bookwyrm/models/site.py

```diff
@@ -209,7 +209,7 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
-def get_passowrd_reset_expiry():
+def get_password_reset_expiry():
```

Comment:
potentially an api break, but I'm hoping it's local...

Reply:
Yes, it's local